aws_database/db_service/models.py

```diff
@@ -0,0 +1,177 @@
+    score = Column(Numeric)
+    flag = Column(Boolean)
+    note = Column(Text)
+    vref = Column(Text, ForeignKey('verseReference.fullverseid'))
+    source = Column(Text)
+    target = Column(JSONB)
+    hide = Column(Boolean)
+    book = Column(Text)
+    chapter = Column(Integer)
+    verse = Column(Integer)
+
+    assessment = relationship("Assessment", back_populates="assessment_results")
+
+class Assessment(Base):
+    __tablename__ = 'assessment'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    revision = Column(Integer, ForeignKey('assessment.revision'))
+    reference = Column(Integer, ForeignKey('assessment.reference'))
+    type = Column(Text)
+    status = Column(Text)
+    requested_time = Column(TIMESTAMP)
+    start_time = Column(TIMESTAMP(timezone=True))
+    end_time = Column(TIMESTAMP(timezone=True))
+
+    assessment_results = relationship("AssessmentResult", back_populates="assessment")
+
+class BibleRevision(Base):
+    __tablename__ = 'bibleRevision'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    date = Column(Date)
+    bibleversion_id = Column(Integer, ForeignKey('bibleVersion.id'), nullable=False)
+    published = Column(Boolean)
+    name = Column(Text)
+    backtranslation_id = Column(Integer, ForeignKey('bibleVersion.id'))
+    machinetranslation = Column(Boolean)
+
+class BibleVersion(Base):
+    __tablename__ = 'bibleVersion'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(Text)
+    isolanguage_id = Column(Text, ForeignKey('isoLanguage.iso639'))
+    isoscript_id = Column(Text, ForeignKey('isoScript.iso15924'))
+    abbreviation = Column(Text)
+    rights = Column(Text)
+    forwardtranslation = Column(Integer)
+    backtranslation = Column(Integer)
+    machinetranslation = Column(Boolean)
+
+class BookReference(Base):
+    __tablename__ = 'bookReference'
+    
+    abbreviation = Column(Text, primary_key=True, index=True)
+    name = Column(Text)
+    number = Column(Integer)
+
+class ChapterReference(Base):
+    __tablename__ = 'chapterReference'
+    
+    fullchapterid = Column(Text, primary_key=True, index=True)
+    number = Column(Integer)
+    bookreference = Column(Text, ForeignKey('bookReference.abbreviation'), nullable=False)
+class IsoLanguage(Base):
+    __tablename__ = 'isoLanguage'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    iso639 = Column(Text, unique=True)
+    name = Column(Text)
+
+class IsoScript(Base):
+    __tablename__ = 'isoScript'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    iso15924 = Column(Text, unique=True)
+    name = Column(Text)
+
+class LocalizedBookName(Base):
+    __tablename__ = 'localizedBookName'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(Text)
+    bookreference_id = Column(Integer, ForeignKey('bookReference.number'))
+    bibleversion_id = Column(Integer, ForeignKey('bibleVersion.id'))
+
+class Question(Base):
```

Comment:
I'm not actually sure what this table was supposed to be for. I think we can remove it.

Reply:
sounds good